Shared/Tests.Shared/PropertyChangedTests.cs

```diff
@@ -36,6 +39,8 @@ public class PropertyChangedTests
 
         private Realm _realm => _lazyRealm.Value;
 
+        private static readonly Task _completedTask = Task.FromResult<object>(null);
```

Comment:
Apf... Should we move that to a `TaskHelper.CompletedTask` to be used in other places as well?